ingestors/drupal.go

```diff
@@ -0,0 +1,125 @@
+package ingestors
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+)
+
+const drupalSchedule = "*/5 * * * *"
+const drupalModulesUrl = "https://www.drupal.org/project/project_module?page=%d&solrsort=ds_project_latest_release+desc"
+const drupalReleasesUrl = "https://www.drupal.org/node/%s/release/feed"
```

Comment:
line 17: first we'll scrape the list of package while we see packages that have fresh releases
line 18: and for each package we get its RSS feed of releases, and ingest fresh releases